test/mongo/topology_test.exs

```diff
@@ -1,6 +1,6 @@
 defmodule Mongo.TopologyTest do
   use ExUnit.Case # DO NOT MAKE ASYNCHRONOUS
-  alias Mongoman.{ReplicaSet, ReplicaSetConfig}
+  # alias Mongoman.{ReplicaSet, ReplicaSetConfig}
```

Comment:
If there do not need to be aliased, I would prefer it be removed instead of just being commented out.